mod_ci/controllers.py

```diff
@@ -133,264 +131,241 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
+                )).first()
                 correct = etree.SubElement(file_node, 'correct')
-                # Need a path that is relative to the folder we provide
-                # inside the CI environment.
+                # Need a path that is relative to the folder we provide inside the CI environment.
                 if last_commit_files is None:
                     correct.text = output_file.filename_correct
                 else:
-                    correct.text = output_file.create_correct_filename(
-                        last_commit_files[0])
+                    correct.text = output_file.create_correct_filename(last_commit_files[0])
+
                 expected = etree.SubElement(file_node, 'expected')
-                expected.text = output_file.filename_expected(
-                    regression_test.sample.sha)
+                expected.text = output_file.filename_expected(regression_test.sample.sha)
+
         # Save XML
         single_test.getroottree().write(
-            os.path.join(base_folder, file_name),
-            encoding='utf-8', xml_declaration=True, pretty_print=True
+            os.path.join(base_folder, file_name), encoding='utf-8', xml_declaration=True, pretty_print=True
         )
         # Append to collection file
         test_file = etree.SubElement(multi_test, 'testfile')
         location = etree.SubElement(test_file, 'location')
         location.text = file_name
+
     # Save collection file
     multi_test.getroottree().write(
-        os.path.join(base_folder, 'TestAll.xml'),
-        encoding='utf-8', xml_declaration=True, pretty_print=True
+        os.path.join(base_folder, 'TestAll.xml'), encoding='utf-8', xml_declaration=True, pretty_print=True
     )
 
     # 2) Create git repo clone and merge PR into it (if necessary)
     try:
-        repo = Repo(os.path.join(
-            config.get('SAMPLE_REPOSITORY', ''), 'vm_data', kvm_name,
-                    'unsafe-ccextractor'))
+        repo = Repo(os.path.join(config.get('SAMPLE_REPOSITORY', ''), 'vm_data', kvm_name, 'unsafe-ccextractor'))
     except InvalidGitRepositoryError:
-        log.critical("[{platform}] Could not open CCExtractor's repository "
-                     "copy!".format(platform=platform))
+        log.critical("[{platform}] Could not open CCExtractor's repository copy!".format(platform=platform))
         return
+
     # Return to master
     repo.heads.master.checkout(True)
     # Update repository from upstream
     try:
         origin = repo.remote('origin')
     except ValueError:
-        log.critical('[{platform}] Origin remote doesn\'t exist!')
+        log.critical("[{platform}] Origin remote doesn't exist!".format(platform=platform))
         return
+
     fetch_info = origin.fetch()
     if len(fetch_info) == 0:
-        log.warn('[{platform}] No info fetched from remote!')
+        log.info('[{platform}] Fetch from remote returned no new data...'.format(platform=platform))
```

Comment:
Probably `"` instead of `'` for consistency.

Reply:
That's some nitpicking that I'd rather do once we have nothing else left to correct, but I agree. It's inconsistent over multiple places.

Reply:
😂  👍